paper-tabs.html

```diff
@@ -200,7 +200,8 @@
     is: 'paper-tabs',
 
     behaviors: [
-      Polymer.IronResizableBehavior
+      Polymer.IronResizableBehavior,
+      Polymer.IronSelectableBehavior,
```

Comment:
Elements team did not end up agreeing with trailing comma on lists of things. Our convention is to omit the comma.